muckrock/templates/lib/foia_log.html

```diff
@@ -7,10 +7,13 @@
     <div class="foia-log__header__right">
       <p class="foia-log__date"><strong>Sent</strong> <time datetime="{{ foia_log.date_requested|date:"c" }}" title="{{ foia_log.date_requested|date:"c" }}">{{foia_log.date_requested}}</time></p>
       {% if foia_log.date_completed %}<p class="foia-log__date"><strong>Completed</strong> <time datetime="{{ foia_log.date_completed|date:"c" }}" title="{{ foia_log.date_completed|date:"c" }}">{{foia_log.date_completed}}</time></p>{% endif %}
+      {% if foia_log.request_url %}
+          <p class="foia-log__request-url"> <strong>Link</strong> <a href="{{ foia_log.request_url }}">Original Request</a></p>
+        {% endif %}
     </div>
   </header>
   <main class="foia-log__main">
-    <p class="foia-log__subject">{{ foia_log.subject|linebreaksbr|urlize }}</p>
+    <p class="foia-log__subject">{{ foia_log.subject }}</p>
```

Comment:
I think the formatting still looks better with the linebreaks in, and urlize seems like a good idea if there are other URLs in the text.

This might have been added in after you branched, you can rebase onto dev to pick it up in that case